### ISSUE TITLE
Not enough filedescriptors for kiwix-serve on library.kiwix.org

### ISSUE THREAD

**Issue by kelson42** [kiwix/maintenance#148]
As a consequence, not all ZIM files can be open. See error:
```
Internal Server Error

An internal server error occured. We are sorry about that :/

error 24 opening file "/var/www/library.kiwix.org/zim/wiktionary/wiktionary_ru_all_maxi_2020-07.zim
```

I is not clear to me exactly where is the problem and why it appears now.

**Comment by kelson42** [comment]
Fixes by #149

**Comment by kelson42** [comment]
@rgaudin Looks like the problem is still there http://library.kiwix.org/eu4_en_all_maxi_2020-08!?
